astroid/nodes/scoped_nodes/mixin.py

```diff
@@ -50,6 +50,24 @@ def scope(self: _T) -> _T:
         """
         return self
 
+    def scope_lookup(self, node, name: str, offset: int = 0):
```

Comment:
```suggestion
    def scope_lookup(self, node: NodeNG, name: str, offset: int = 0) -> tuple(str, list(NodeNG):
```

Reply:
I'm not sure this is actually true. Wherever `node: NodeNG` I have become real suspicious that it should be `node: NodeNG | Instance`.
I'll investigate it later, `mypy` at least passes now.

That's also why I let the docstring in.